charts/edge/README.md

```diff
@@ -29,20 +29,22 @@ helm install edge mezmo/edge \
-| pvcName                   | string   | ""              | Optional name for an existing PersistentVolumeClaim to use for disk buffering, aggregate persistence and k8s log checkpoints. Leaving undefined will default to ephemeral storage lasting the lifetime of the pod
-| maxDiskBufferBytes        | int      |                 | Optional max number of bytes to store in a disk-backed buffer. Leaving undefined will use default value of 268435488 (256MiB)
-| maxMemBufferEvents        | int      |                 | Optional max number of events to store in a memory buffer. Leaving undefined will use default value of 1000
+| **Key**                      | **Type** | **Default**     | **Description**
+| ---------------------------- | -------- | --------------- | ----------------------------------------------------------------------
+| mezmoApiAccessToken          | string   |                 | Your Mezmo API access token
+| mezmoDeploymentGroup         | string   |                 | Deployment group to apply to this Edge instance. Leaving undefined pulls all Edge pipelines for the org
+| mezmoApiScheme               | string   | "https"         | The scheme to use for the Mezmo API URL
+| mezmoApiHost                 | string   | "api.mezmo.com" | The hostname(:port) of the Mezmo API
+| logLevel                     | string   | info            | Controls the logging verbosity of the deployment
+| autoscaling.enabled          | boolean  | false           | Whether or not to enable a HorizontalPodAutoscaler for this deployment
+| service.sourcePorts.start    | int      | 8000            | The start of the port range (inclusive [start, end])
+| service.sourcePorts.end      | int      | 8010            | The end of the port range (set 0 or "" to disable port range)
+| service.sourcePorts.list     | array[]  | []              | Optional list of discrete ports to configure on the service
+| enableK8sLogs                | boolean  | false           | Whether or not to add ClusterRole and Volume access required for k8s logs source
+| pvcName                      | string   | ""              | Optional name for an existing PersistentVolumeClaim to use for disk buffering, aggregate persistence and k8s log checkpoints. Leaving undefined will default to ephemeral storage lasting the lifetime of the pod
+| maxDiskBufferBytes           | int      |                 | Optional max number of bytes to store in a disk-backed buffer. Leaving undefined will use default value of 268435488 (256MiB)
+| maxMemBufferEvents           | int      |                 | Optional max number of events to store in a memory buffer. Leaving undefined will use default value of 1000
+| reduceByteThresholdPerState  | int      |                 | Optional max number of bytes to store in a memory for any given reduce component. Leaving undefined will use default value of 102400 (100Kb)
```

Comment:
Note there is a column for default, so we could remove that bit from the description